src/views/header/RightHeaderBlock.js

```diff
@@ -25,50 +28,65 @@ const View = boneView.extend({
       return this.render();
     });
 
+    // collection view for all the headers
+    this.addView("headers", new (boneView.extend({
```

Comment:
I'm making two views here.
1 for the all previous header components, and another one for the pinned block canvas.
Grouping it like this makes it easier to manage re-renders/dom manipulations.